docs/src/pages/guides/probes.md

```diff
@@ -45,21 +45,24 @@ An actual probe request may be something like below.
           "password": "somepassword"
         }
       },
+      "successThreshold": 3,
+      "failedThreshold": 3,
       "alerts": ["status-not-2xx", "response-time-greater-than-200-ms"]
     },
   ]
 ```
 
 Details of the field are give in the table below.
 
-| Topic   | Description                                                    |
-| :------ | -------------------------------------------------------------- |
-| method  | Http method such as GET, POST, PUT, DELETE                     |
-| url     | This is the url endpoint to dispatch the request to            |
-| timeout | Request timeout, after which time, `Monika`will assume timeout |
-| headers | Your http header                                               |
-| body    | Any http body if your method requires it                       |
-|         |                                                                |
+| Topic                       | Description                                                                                                                    |
+| :-------------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| method                      | Http method such as GET, POST, PUT, DELETE                                                                                     |
+| url                         | This is the url endpoint to dispatch the request to                                                                            |
+| timeout                     | Request timeout, after which time, `Monika`will assume timeout                                                                 |
+| headers                     | Your http header                                                                                                               |
+| body                        | Any http body if your method requires it                                                                                       |
+| successThreshold (optional) | How many times should a probe trigger send notifications and change the server to 'UP' if current server is 'DOWN' (default 5) |
+| failedThreshold (optional)  | How many times should a probe trigger send notifications and change the server to 'DOWN' if current server is 'UP' (default 5) |
```

Comment:
Number of times an alert should return false before Monika sends notifications. For example, when failedThreshold is 3, Monika will only send notifications when the probed URL returns status 2xx 3 times in a row. After sending the notifications, Monika will not send notifications anymore until the alert status changes. Default value is 5.